scripts/verify_multi_user.py

```
@@ -0,0 +1,155 @@
+
+    # 8. Test Unauthorized (No Roles) Permissions
+    guest_name = f"guest_{RUN_ID}"
+    print(f"\nStep 8: Testing Guest ({guest_name}, NO roles) permissions...")
+    
+    # Create Guest
+    httpx.post(f"{BASE_URL}/api/v1/users", headers=admin_headers, json={
+        "username": guest_name, "password": "Password123!"
+    })
+    # Explicitly clear roles (in case default changes)
+    resp_user = httpx.get(f"{BASE_URL}/api/v1/users", headers=admin_headers)
+    guest_id = [u["id"] for u in resp_user.json() if u["username"] == guest_name][0]
```

Ruff (0.14.14)

[warning] Prefer `next(...)` over single element slice

Replace with `next(...)`

(RUF015)